python/fastquant/backtest/backtest.py

```diff
@@ -153,16 +153,24 @@ def backtest(
             )
             strat_names.append(strat)
     else:
+
+        # Allow instance of BaseStrategy or from the predefined mapping
+        if type(strategy) != str:
```

Comment:
```suggestion
        if issubclass(strategy, bt.Strategy):
```

Reply:
It's cleaner if we directly check if the input class is a subclass of `bt.Strategy`, since every backtrader strategy inherits from `bt.Strategy`.

Reply:
@jbdelmundo this also means you'll have to change the line 190 from `if strategy == "sentiment":` to `if strategy_name == "sentiment":`

This should fix the failing tests :smile:

Reply:
Fixed! Thanks!!